app/repositories/user_repository.py

```diff
@@ -86,3 +86,21 @@ async def get_my_requests(self, id: UUID) -> List[UserInvitationSchema]:
         my_requests = await self.db.execute(select(Action).where(Action.user_id == id).where(Action.status == ActionStatus.REQUESTED))
         my_requests = my_requests.scalars().all()
         return my_requests
+    
+    async def get_notifications(self, user_id):
+        notifications = await self.db.execute(select(Notification).where(Notification.user_id == user_id))
+        notifications = notifications.scalars().all()
+        return notifications
+    
+    async def mark_notification_as_read(self, notification_id):
+        notification = await self.db.execute(select(Notification).where(Notification.id == notification_id))
+        notification = notification.scalars().first()
+        notification.status = NotificationStatus.READ
+        self.db.add(notification)
+        await self.db.commit()
+        return notification
```

Comment:
would be helpful to add mark_all_active_notifications_as_read
try to add this method